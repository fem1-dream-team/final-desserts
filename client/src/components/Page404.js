import React from 'react';

export const Page404 = () => {
        return (
            <div>
                Ooops, can't find that page
                ERROR 404
            </div>
        );
};