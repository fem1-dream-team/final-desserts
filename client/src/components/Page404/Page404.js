import React from 'react';
import styled from 'styled-components';
import { NavLink } from 'react-router-dom'

export const Page404 = () => {
	return (
		<Container>
			<h1>Ooops, can't find that page...</h1>
			<span>Error 404</span>
			<LinkDiv to='/'><DivButton>CONTINUE BROWSING HERE</DivButton></LinkDiv>
		</Container>
	)
}

const Container = styled.div`
display: flex;
flex-direction: column;
margin: 0 auto;
color: black; 
font-size: 16px;
text-align: center;
vertical-align: center;
`
const DivButton = styled.div`
	width: 250px;
	margin: 35px auto;
	padding: 15px 20px;
	color: white;
	background-color: #f186a0;
	border-radius: 10px;
`
const LinkDiv = styled(NavLink)`
	text-decoration: none;
`