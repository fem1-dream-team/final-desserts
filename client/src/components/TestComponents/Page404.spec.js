import React from "react";
import { Page404 } from "../Page404/Page404";
import renderer from 'react-test-renderer'

describe("Page404 component", () => {

	it("Page404 component render", () => {
		const wrapper = renderer.create(<Page404 />).toJSON();
		expect(wrapper).toMatchSnapshot();

	});
});