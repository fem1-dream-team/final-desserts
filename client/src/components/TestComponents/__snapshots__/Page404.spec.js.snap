// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404 component Page404 component render 1`] = `
<div>
  Ooops, can't find that page. ERROR 404
</div>
`;
